lib/components/narrative/narrative-itineraries-header.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react'
+import styled from 'styled-components'
+import { FormattedMessage, useIntl } from 'react-intl'
+
+import Icon from '../util/icon'
+
+import PlanFirstLastButtons from './plan-first-last-buttons'
+import SaveTripButton from './save-trip-button'
+
+const IssueButton = styled.button`
+  background-color: #ecbe03;
+  border: none;
+  border-radius: 5px;
+  display: inline-block;
+  font-size: 12px;
+  padding: 2px 4px;
+`
+
+export default function NarrativeItinerariesHeader({
+  errors,
+  itineraries,
+  itineraryIsExpanded,
+  onSortChange,
+  onSortDirChange,
+  onToggleShowErrors,
+  onViewAllOptions,
+  pending,
+  showingErrors,
+  sort
+}: {
+  errors: unknown[]
+  itineraries: unknown[]
+  itineraryIsExpanded: boolean
+  onSortChange: () => void
+  onSortDirChange: () => void
+  onToggleShowErrors: () => void
+  onViewAllOptions: () => void
+  pending: boolean
+  showingErrors: boolean
+  sort: { direction: string; type: string }
+}): JSX.Element {
+  const intl = useIntl()
+
+  return (
+    <div
+      className="options header"
+      style={{
+        alignItems: 'end',
+        display: 'flex',
+        flexWrap: 'wrap'
+      }}
+    >
+      {itineraryIsExpanded || showingErrors ? (
+        <>
+          <button
+            className="clear-button-formatting"
+            onClick={onViewAllOptions}
+          >
+            <Icon type="arrow-left" />{' '}
```

Comment:
Oops this one fell off the cracks, replace `{' '}` by writing `<Icon type=... withSpace />`.